### ISSUE TITLE
After upgrade to angular 15 with nx 16 compiling is happening non stop without making changes

### ISSUE THREAD

**Issue by DimitriLib** [nrwl/nx#17272]
### Current Behavior

After upgrading nx and angular application keep recompiling when running ng serve. First time ng serve works without recompiling. Once change is made it conitinues recompiling of unchanged code until runs out of memory.
It seems that issue due to project.json file having assets that points to css and other files in node-module directory.
          {
            "glob": "**/*",
            "input": "./node_modules/tinymce",
            "output": "/tinymce/"
          }

### Expected Behavior

App should only recompile when changes made to application or library.

### GitHub Repo

https://github.com/DimitriLib/nx-angular-upgrade13-14

### Steps to Reproduce
ng serve test
make change to any scss code in the repository and save it. after that recompiling would start and keep going until run out of memory.

### Nx Report

```shell
Node   : 18.12.1                                         
   OS     : win32 x64                                       
   npm    : 8.19.2
   Hasher : Native

   nx                 : 16.0.1
   @nx/js             : 16.0.1
   @nx/jest           : 16.0.1
   @nx/linter         : 16.0.1
   @nx/workspace      : 16.0.1
   @nx/angular        : 16.0.1
   @nx/cypress        : 16.0.1
   @nx/devkit         : 16.0.1
   @nx/eslint-plugin  : 16.0.1
   @nx/storybook      : 16.1.4
   @nrwl/tao          : 16.0.1
   @nx/webpack        : 16.0.1
   nx-cloud           : 16.0.5
   typescript         : 4.9.5
   ---------------------------------------
   Community plugins:
   @ngrx/effects              : 15.3.0
   @ngrx/store                : 15.3.0
   @ngrx/store-devtools       : 15.3.0
   @trellisorg/make-buildable : 0.0.11
   @compodoc/compodoc         : 1.1.19
   @storybook/angular         : 7.0.10
   @twittwer/compodoc         : 1.8.2
   ---------------------------------------
   The following packages should match the installed version of nx
     - @nx/storybook@16.1.4
     - @nrwl/storybook@16.1.4

   To fix this, run `nx migrate nx@16.1.4`
```

### Failure Logs

```shell
No changes but keep recompiling:

√ Compiled successfully.
√ Browser application bundle generation complete.

16 unchanged chunks

Build at: 2023-05-28T05:55:44.615Z - Hash: 34cb661c99589867 - Time: 1008ms

√ Compiled successfully.
√ Browser application bundle generation complete.

16 unchanged chunks

Build at: 2023-05-28T05:55:46.066Z - Hash: 34cb661c99589867 - Time: 1082ms

√ Compiled successfully.
√ Browser application bundle generation complete.

16 unchanged chunks

Build at: 2023-05-28T05:55:47.286Z - Hash: 34cb661c99589867 - Time: 1106ms
```

### Operating System

- [ ] macOS
- [ ] Linux
- [X] Windows
- [ ] Other (Please specify)

**Comment by leosvelperez** [comment]
Thanks for reporting this!

Unfortunately, we can't reproduce the issue. Please provide a reproduction so we can troubleshoot this or at the very least, detailed reproduction steps that can be followed to create a workspace where the issue happens.

**Comment by DimitriLib** [comment]
@leosvelperez added repo and steps. it seems that assets in project.json causing recompilation. The same code was working in vesion 14 of angular and nx 15.  Can you please check?

**Comment by leosvelperez** [comment]
@DimitriLib thanks for providing a repro! Unfortunately, I still can't reproduce it though I'm not on a Windows machine. I'll try to get someone with one to reproduce it and confirm whether it is only a Windows issue.

**Comment by Coly010** [comment]
I couldn't reproduce this on Windows either.

The nx report I receive from within your repo:

```
 >  NX   Report complete - copy this into the issue template

   Node   : 18.16.0
   OS     : win32 x64
   npm    : 9.5.1
   Hasher : Native

   nx (global)        : 16.2.2
   nx                 : 16.0.1
   @nx/js             : 16.0.1
   @nx/jest           : 16.0.1
   @nx/linter         : 16.0.1
   @nx/workspace      : 16.0.1
   @nx/angular        : 16.0.1
   @nx/cypress        : 16.0.1
   @nx/devkit         : 16.0.1
   @nrwl/devkit       : 14.8.8
   @nx/eslint-plugin  : 16.0.1
   @nx/storybook      : 16.1.4
   @nrwl/tao          : 16.0.1
   @nx/webpack        : 16.0.1
   nx-cloud           : 16.0.5
   typescript         : 4.9.5
   ---------------------------------------
   Community plugins:
   @ngrx/effects              : 15.3.0
   @ngrx/store                : 15.3.0
   @ngrx/store-devtools       : 15.3.0
   @trellisorg/make-buildable : 0.0.11
   @compodoc/compodoc         : 1.1.19
   @storybook/angular         : 7.0.18
   @twittwer/compodoc         : 1.8.2
   ---------------------------------------
   The following packages should match the installed version of nx
     - @nrwl/devkit@14.8.8
     - @nx/storybook@16.1.4
     - @nrwl/storybook@16.1.4

   To fix this, run `nx migrate nx@16.1.4`
```

The only difference I'm seeing really is version of Node. 

This isn't an Nx issue and is likely related to something in your machine's environment unfortunately
